src/py/covidr/config/development.py

```diff
@@ -15,3 +15,8 @@ class DevelopmentDatabaseConfig(DatabaseConfig):
     @property
     def URI(self):
         return "postgresql://user_rw:password_rw@localhost:5432/covidr_db"
+
+    # ensures that latest static assets are read during frontend dev work
+    @property
+    def SEND_FILE_MAX_AGE_DEFAULT(self):
+        return 0
```

Comment:
python's doc blocking style is _weird_

```suggestion

    @property
    """Ensures that latest static assets are read during frontend dev work."""
        def SEND_FILE_MAX_AGE_DEFAULT(self):
        return 0
```

Reply:
Got it, thanks